config/app.js

```diff
@@ -1,4 +1,6 @@
 const { joinPath } = require('../lib/utils')
+require('dotenv').config({path: './.env.app'})
```

Comment:
Is this correct place to require dotenv? The variables below seem to change correctly when placed in .env.app file.